pkgs/development/python-modules/molecule/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ansible-lint
+, cerberus
+, click
+, click-help-colors
+, click-completion
+, cookiecutter
+, pluggy
+, pyyaml
+, selinux-python
+, subprocess-tee
+}:
+
+buildPythonPackage rec {
+  pname = "molecule";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9ac0fde11f895bbc6727de26e2ab0bd0316964f4e4facfd821240959814f9572";
+  };
+
+  propagatedBuildInputs = [
+    ansible-lint
+    cerberus
+    click
+    click-completion
+    click-help-colors
+    cookiecutter
+    pluggy
+    pyyaml
+    selinux-python
+    subprocess-tee
+  ];
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  checkPhase = "true";
```

```suggestion
  dontCheck = false;
```

Why are those disabled? Please add an pythonImportsCheck instead.